demo-shell-ng2/app/components/activiti/activiti-demo.component.html

```diff
@@ -85,7 +85,8 @@
                     <div class="mdl-cell mdl-cell--10-col task-column mdl-shadow--2dp" *ngIf="isStartTaskMode()">
                          <activiti-start-task
                                 [appId]="appId"
-                                (onSuccess)="onStartTaskSuccess($event)">
+                                (onSuccess)="onStartTaskSuccess($event)"
```

Comment:
can you change from onSuccess to success (do not forget to update the documentation as well)